SmartVision/WindowsForms/MissingPerson.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WindowsForms
+{
+    class MissingPerson
+    {
+        string name;
+        string surname;
+        string description;
+        DateTime birthday;
+        Image photo;
```

Comment:
Maybe we also should have a contact email or smth. to ping, when a missing person is found? We can implement it later, but there will be less files to change if we add it now

Reply:
Well the missing person is probably not the person we gonna ping to contact, it should be the user who adds the person, so kinda different classes. But yes a good idea for the future.

Reply:
Ok. So in the future we'll need to make a reference to another class, smth like ContactPerson,